Cargo.toml

```diff
@@ -1,13 +1,6 @@
 [workspace]
-members = [
-  "oak/server/rust/oak_loader",
-  "oak/server/rust/oak_runtime",
-  "runner",
-  "sdk/rust/oak",
-  "sdk/rust/oak_tests",
-  "third_party/roughenough",
-]
-exclude = ["oak_abi", "oak_utils"]
+members = ["oak/server/rust/oak_loader", "oak/server/rust/oak_runtime"]
```

Comment:
It feels we should just move this file under oak/server?

Reply:
I thought this change might break too many things and would better to be done in a separate PR. But I can give it a try here :)